zebra-chain/src/transparent.rs

```diff
@@ -454,6 +460,22 @@ impl Output {
             }
         }
     }
+
+    /// Returns true if this output is considered dust.
+    pub fn is_dust(&self) -> bool {
+        let output_size: u32 = self
+            .zcash_serialize_to_vec()
+            .expect("serializing to vec should not fail")
+            .len()
+            .try_into()
+            .expect("output size should fit in u32");
```

Comment:
This is a bit more efficient:

```suggestion
        let output_size: u32 = self
            .zcash_serialized_size()
            .try_into()
            .expect("output size should fit in u32");
```